samples/unit-tests/breadcrubs/breadcrumbs-in-treemap/demo.html

```diff
@@ -0,0 +1,10 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/modules/data.js"></script>
+<script src="https://code.highcharts.com/modules/heatmap.js"></script>
+<script src="https://code.highcharts.com/modules/treemap.js"></script>
+<script src="https://code.highcharts.com/modules/breadcrumbs.js"></script>
```

Comment:
Breadcrumbs should not be a separate module in the distribution. It should be a separate _ES module_, but imported into the drilldown and treemap distribution modules/JS files.